### PR TITLE
doc: refine fscache user guide with containerd runtime

### PR DESCRIPTION
**Pull request by hsiangkao**
Signed-off-by: Gao Xiang \<hsiangkao@linux.alibaba.com\>

### CODE REVIEW

File: docs/nydus-fscache.md [hunk 1]
```diff
@@ -85,20 +92,49 @@ For more information on how to configure containerd to use nydus snapshotter ple
...
+7. Run container with [ctr-remote](../contrib/ctr-remote)
+
+``` shell
 # pull nydus image
-ctr-remote images rpull hsiangkao/ubuntu:20.04-rafs-v6-docker
+contrib/ctr-remote/bin/ctr-remote images rpull docker.io/hsiangkao/ubuntu:20.04-rafs-v6-docker
 
 # run nydus image
-ctr-remote run --rm -t --snapshotter=nydus hsiangkao/ubuntu:20.04-rafs-v6-docker ubuntu /bin/bash
+ctr run --rm -t --snapshotter=nydus docker.io/hsiangkao/ubuntu:20.04-rafs-v6-docker ubuntu /bin/bash
```
Comment (imeoer) [inline docs/nydus-fscache.md:131]: We can also run with `ctr-remote`, because `ctr-remote` has already been compiled, so just use it.

Comment (hsiangkao) [reply docs/nydus-fscache.md:131]: ctr doesn't installed by default? I think using the default tool is always perferred, no?

Comment (imeoer) [reply docs/nydus-fscache.md:131]: Well, if we assume that `ctr` is already installed, we can use it. `ctr-remote` is actually the same as ctr except a new `rpull` subcommand.

Comment (hsiangkao) [reply docs/nydus-fscache.md:131]: Yeah, I agree, is there some cases that `ctr` didn't install together with `containerd`?

**Pull request merged**